proto/serving.proto

```diff
@@ -64,3 +66,25 @@ message Value {
         bytes on_demand_function = 8;
     }
 }
+
+message SourceID {
+  string name = 1;
+  string version = 2;
```

Comment:
variant? to keep things consistent?

Reply:
`version` actually is the consistent naming convention for the protos. I figured it would be better to refactor them later together instead of introducing this inconsistency.